app/utils/k-formatter.js

```diff
@@ -0,0 +1,3 @@
+export const kFormatter = (num) => {
```

Comment:
We don't need a kFormatter. What would happen if you have millions or billions? It's better if we use a number formatter library. As we have already installed `d3` in the app we have access to d3-format

Reply:
```
import { format } from 'd3-format';

format('.3s')(value)
```